templates/taxi/driver_detail.html

```diff
@@ -4,6 +4,12 @@
   <h1>
     Username: {{ driver.username }}
   </h1>
+  <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
+    Delete
+  </a>
+  <a href="{% url 'taxi:driver-update' pk=driver.id %}" class="btn btn-primary link-to-page">
+    Update License number
+  </a>
```

Comment:
Consider adding confirmation dialogs for the delete and update actions to prevent accidental operations. This can enhance user experience by ensuring that users are certain about their actions.